app/graphql/level-component.tsx

```diff
@@ -0,0 +1,42 @@
+import { gql } from "@apollo/client"
+import * as React from "react"
+import { PropsWithChildren } from "react"
+import { useLevelQuery } from "./generated"
+import { useIsAuthed } from "./is-authed-context"
+import { LevelContextProvider } from "./level-context"
+
+gql`
+  query network {
+    globals {
+      network
+    }
+  }
+
+  query level {
+    me {
+      id
+      defaultAccount {
+        id
+        level
+      }
+    }
+  }
+`
+
+export const LevelContainer: React.FC<PropsWithChildren> = ({ children }) => {
```

Comment:
Whats the thinking for adding a context vs just making the gql query directly in components?